jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -185,6 +186,19 @@ function configure_workbench_profile() {
     fi
 }
 
+# Checks if the know_hosts is present when id_rsa is provided
+checkSSHFiles(){
+    local id_rsa=/home/jboss/.ssh/id_rsa
+    local know_hosts=/home/jboss/.ssh/known_hosts
+    if [ -f ${id_rsa} ]; then
+        if [ -f ${know_hosts} ]; then
+            log_info "/home/jboss/.ssh/id_rsa and /home/jboss/.ssh/known_hosts provided."
+        else
+            log_error "/home/jboss/.ssh/id_rsa is present but /home/jboss/.ssh/known_hosts wasn't provided, will be used known_hosts in /etc/ssh/known_hosts"
```

Comment:
My understanding is that this file is empty and jboss user does not have permission to write on /etc/ssh.

Reply:
Yep, maybe the msg could be "/home/jboss/.ssh/id_rsa is present but /home/jboss/.ssh/known_hosts wasn't provided, please redeploy and provide id_rsa and known_hosts"

Reply:
so, what's to point to add the known_hosts on /etc?